WSLBlobNFS/WSLBlobNFS.psm1

```diff
@@ -149,6 +149,10 @@ function Install-WSLBlobNFS-Internal
     {
         Write-Output "WSL is not installed. Installing WSL..."
 
+        # For WSL1 users, we need to update to WSL2 before we install the distro.
+        wsl --update | Out-Null
+        wsl --set-default-version 2 | Out-Null
```

Comment:
you said it was failing? why did you add it here then?
also, with this here, do we need L159 and 161